kelo_hitbot_driver/MotionCommand command
int32 ovl
float32 blendTime
uint8 offsetFlag
kelo_hitbot_driver/Axes offset

